.github/workflows/kickstart-tests.yml.j2

```diff
@@ -73,12 +73,28 @@ jobs:
           echo "workflow arguments are: $ARGS"
           echo "comment_args=${ARGS}" >> $GITHUB_OUTPUT
 
+      - name: Get timestamp
+        id: timestamp
+        run: echo "::set-output name=timestamp::$(date +'%Y-%m-%d_%H:%M:%S')"
+
+      - name: Construct image description
+        id: kstest_run_description
+        run: |
+          set -eux
+          # comment on PR
+          pr_num="${{ github.event.issue.number }}"
+          timestamp=${{ steps.timestamp.outputs.timestamp }}
+          sha="${{ steps.pr_api.outcome == 'success' && fromJson(steps.pr_api.outputs.data).head.sha }}"
```

Comment:
This shows up as boolean in the image description. I guess you can remove the success part.